LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017-2019 Rafael Beraldo
+Copyright (c) 2017-2024 [these people](https://github.com/buefy/buefy/graphs/contributors)
```

Comment:
@wesdevpro We have to append the file extension `.md` to make this work.
`LICENSE` → `LICENSE.md`